docs/core/porting/index.md

```diff
@@ -13,35 +13,91 @@ You might have code that currently runs on the .NET Framework that you're intere
 
 ## Overview of the porting process
 
+Porting to .NET Core (or .NET Standard) from .NET Framework can become complicated very quickly depending on the complexity of the project. This document should help with identifying the main strategies that have been employed by users to successfully convert their code bases. This will address the conversion at two levels: solution-wide and project specific.
+
+Each of these steps introduce potential places for behavior changes, so ensure that you adequately test your library or application before continuing on to later steps. The first steps are to get your project ready for a switch to .NET Standard/.NET Core. If you have unit tests, it is best to convert them first so that you can continue testing changes in the product you are working on. Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
```

Comment:
I must say, this sounds a bit scary :) Depending on the scenario, migration *can* be a lot of work. But other scenarios (libraries and desktop apps, specifically) often port over without too much pain. We may want to rephrase a bit to not be too intimidating.